products/compute/terraform.yaml

```diff
@@ -919,6 +919,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     exclude: true
   InstanceGroupManager: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  InstanceGroupNamedPort: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: 'projects/{{project}}/zones/{{zone}}/instanceGroups/{{group}}/{{port}}/{{name}}'
```

Comment:
I'd also add a mutex for the instance group so multiple named ports don't get added all at the same time